udata_front/theme/gouvfr/templates/macros/report.html

```diff
@@ -1,6 +1,6 @@
 {% macro report_btn(obj) %}
 {% set object_type = obj.__class__.__name__ %}
-{% if (obj.organization and obj.organization.certified) or obj.certified %}
+{% if (obj.organization and not obj.organization.certified) or not obj.certified %}
```

Comment:
I think `obj.certified` does not exist on reuses or datasets, making `not obj.certified` true too often. This should only apply on organization page?

Reply:
Yes, the `or` is for the organization page

Reply:
But then we show the report button on dataset published by certified organization, when we should not right?

Reply:
Indeed, my condition is wrong